datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx

```diff
@@ -10,7 +10,6 @@ import { EntityAndType } from '../../../types';
 
 const SearchBody = styled.div`
     height: 100%;
-    overflow-y: scroll;
```

Comment:
removing this CSS makes me nervous as this will affect many other places in the code. It appears that the only place affected by this weird double scroll situation is the glossary related entity tab, while other places using this component are totally fine.

It's because of that that I would suggest changing `GlossaryRelatedEntity` instead of changing this component that usually looks good.

I tested locally and if you simply change the return of `GlossaryRelatedEntity.tsx` from:
```
    return (
        <GroupAssetsWrapper>
            <Col md={24} lg={24} xl={24}>
                <EmbeddedListSearchSection
                    fixedQuery={fixedQueryString}
                    emptySearchQuery="*"
                    placeholderText="Filter entities..."
                />
            </Col>
        </GroupAssetsWrapper>
    );
```
to:
```
    return (
        <EmbeddedListSearchSection
            fixedQuery={fixedQueryString}
            emptySearchQuery="*"
            placeholderText="Filter entities..."
        />
    );
```
your problem is fixed